docs/repobee_plug/overview.rst

```diff
@@ -168,6 +167,4 @@ complete example of a plugin written with this approach, see the
 .. _javac plugin: https://github.com/repobee/repobee/blob/master/repobee/ext/javac.py
 .. _pylint plugin: https://github.com/repobee/repobee/blob/master/repobee/ext/pylint.py
 
-.. _repobee built-ins: https://repobee.readthedocs.io/en/stable/plugins.html#built-in-plugins
 .. _repobee-junit4: https://github.com/repobee/repobee-junit4
-.. _Using Existing Plugins: https://repobee.readthedocs.io/en/stable/plugins.html#using-existing-plugins
```

Comment:
The removed lines were never mentioned anywhere and also this formatting is no longer supported due to the bundling